src/pages/en/guides/markdown-content.md

```diff
@@ -96,7 +96,7 @@ const { frontmatter, url } = Astro.props;
 ---
 <html>
   <head>
-    <meta rel="canonical" href={new URL(url, Astro.site).pathname}>
+    <link rel="canonical" href={new URL(url, Astro.site).pathname}>
```

Comment:
I actually think we’re safer to remove this entirely. It’s not an essential part of this example, and we don’t want to distract from the main focus of showing how layouts work by needing to explain this detail.

```suggestion
```